README.md

```diff
@@ -623,11 +755,16 @@ function vendingMachine (snackSelection, moneyInserted) {
 }
 ```
 
-We removed the `default` case from `vendingMachine()`. In its place, we added a guard at the top of the function that exits immediately if the `snackSelection` is invalid, avoiding the expense of having to enter the `switch` statement. If the selection is valid, `validateSelection()` returns `true`. If the selection is invalid, `validateSelection()` returns `false`. Simple as that! This change actually **added** a bit more code, but that's far outweighed by the benefits:
+We removed the `default` case from `vendingMachine()`. In its place, we added a
+guard at the top of the function that exits immediately if the `snackSelection`
+is invalid, avoiding the expense of having to enter the `switch` statement. If
+the selection is valid, `validateSelection()` returns `true`. If the selection
+is invalid, `validateSelection()` returns `false`. Simple as that! This change
+actually **added** a bit more code, but that's far outweighed by the benefits:
```

Comment:
`*added*` so that it's italicized versus bolded (I'm not 100% sure how consistent or emphasis is otherwise, but bold seems to be best for IMPORTANT information, versus emphasized).